File: Sdk-ObjectiveC/office365-base-sdk/office365-base-sdk/BaseClient.m
//
//  BaseClient.m
//  office365-base-sdk
//
//  Created by Gustavo on 7/17/14.
//  Copyright (c) 2014 Lagash. All rights reserved.
//

#import "BaseClient.h"

@implementation BaseClient

- (id)initWithUrl:(NSString *)url credentials:(Credentials *)credentials
{
    self.Credential = credentials;
    self.Url = url;
    return self;
}

- (NSMutableArray *)parseData:(NSData *)data{
    return [NSMutableArray alloc];
}

@end

File: Sdk/office365_base_sdk/office365_base_sdk/BaseClient.swift
//
//  BaseClient.swift
//  office365-base-sdk
//
//  Created by Lagash on 6/17/14.
//  Copyright (c) 2014 microsoft. All rights reserved.
//

import Foundation

class BaseClient : NSObject{
    
    var Credential : Credentials;
    var Url : NSString;
    
    init(url : NSString ,credentials : Credentials){
        
        self.Credential = credentials;
        self.Url = url;
    }
    
    var data: NSMutableData = NSMutableData();
    
    func parseData(data : NSData) -> NSMutableArray{

        return NSMutableArray();
    }
}